qubit-sdk/src/main/java/com/qubit/android/sdk/internal/eventtracker/EventTrackerImpl.java

```diff
@@ -318,6 +325,7 @@ public void run() {
       LOGGER.d("SendEventTask: Sending events: " + eventRestModels.size() + ", dedupe=" + dedupe);
       EventsRestAPIConnector.ResponseStatus status = apiConnector.sendEvents(eventRestModels, dedupe);
       LOGGER.d("SendEventTask: Events sent. Status: " + status);
+      storeEventsToSend(eventsToSend);
```

Comment:
I don't understand the idea of placements, but the place where it stored is suspicious to me.

Why is it stored here? 
I see two better places:
- when event is added to queue
- when event successfully leaves the queue

Reply:
Moved to `StoreEventTask`. 

Requirement here is that we shouldn't worry whether event was successfully sent/received by the API. If it has been at least attempted to be sent that's enough for it to be cached.